### PR TITLE
borgbackup: updated to 1.1.5; generic service

### PR DESCRIPTION
**Pull request by yuroller**
_Motivation:_ Update borgbackup to 1.1.5
_Linked issues:_ #3196 #2998 #2351

### Checklist
- [] Build rule `all-supported` completed successfully
- [] Package upgrade completed successfully
- [*] New installation of package completed successfully

Little testing on synology DS212+ DSM 6.0.2

### CODE REVIEW

File: spk/borgbackup/src/service-setup.sh [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-preinst ()
-{
-    exit 0
-}
-
-postinst ()
+service_postinst ()
 {
     # Link
     ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
```
Comment (ymartin59) [inline spk/borgbackup/src/service-setup.sh]: This is already done in generic `installer.sh`

File: spk/borgbackup/src/service-setup.sh [hunk 2]
```diff
@@ -30,29 +23,13 @@ postinst ()
...
-preuninst ()
-{
-    exit 0
 }
 
-postuninst ()
+service_postuninst ()
 {
     # Remove link
     rm -f ${INSTALL_DIR}
```
Comment (ymartin59) [inline spk/borgbackup/src/service-setup.sh]: This is already done in generic `installer.sh`

File: spk/borgbackup/src/service-setup.sh [hunk 3]
```diff
@@ -1,21 +1,14 @@
-#!/bin/sh
-
 # Package
 PACKAGE="borgbackup"
```
Comment (ymartin59) [inline spk/borgbackup/src/service-setup.sh]: May be removed

File: spk/borgbackup/src/service-setup.sh [hunk 4]
```diff
@@ -1,21 +1,14 @@
...
-DNAME="Borg"
+
 
 # Others
 INSTALL_DIR="/usr/local/${PACKAGE}"
```
Comment (ymartin59) [inline spk/borgbackup/src/service-setup.sh]: May be removed and replaced by `${SYNOPKG_PKGDEST}`

**Comment by yuroller** [comment]
I modified the service-setup.sh script, as you suggested.
Then I amended the commit and forced the push.
The single commit pull request should be updated now.

**Comment by ymartin59** [comment]
Yet a last question... do you know if requirement wheels are all architecture independant, which may allow borgbackup to be "no arch", so one single package for all archs ?

**Comment by yuroller** [comment]
borgbackup builds some modules with c compiler, so each architecture needs a specific build

**Pull request merged**